tests/shipment_test.go

```diff
@@ -215,14 +215,15 @@ func (c *ClientTests) TestShipmentSmartrates() {
 	require.NotEmpty(shipment.Rates)
 
 	smartrates, err := client.GetShipmentSmartrates(shipment.ID)
+	require.NoError(err)
 	assert.Equal(shipment.Rates[0].ID, smartrates[0].ID)
-	assert.Equal(smartrates[0].TimeInTransit.Percentile50, 1)
```

Comment:
curious as to why we removed the exact value checks?

these tests are using VCR so these values should remain the same.

Reply:
This is true, though the rate at which we've been re-recording cassettes (we've made various breaking changes across the libraries such as switching from form encoding to JSON encoding so we re-record the whole test suite) is annoying to constantly update these assertions because although VCR keeps the values the same, when removed, these values vary widely across runs and updating 7 entries is taxing. 

We're switching the other libraries to have a looser assertion here as a convenience to faster testing loops.

Reply:
thanks for the explanation.